src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -0,0 +1,1014 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
```

Comment:
I think this should be the only file changed in this PR, right? Is it possible to remove the others (I guess for some you may need to rebase?) so that it's clear what the intent of this PR is?

Reply:
we are working on removing other changes. Will update shortly.

Reply:
we have removed unnecessary changes as suggested.